editor/src/uuiui/inline-button.tsx

```diff
@@ -47,3 +45,29 @@ export const InlineButton = styled.button({
     outline: 'none',
   },
 })
+
+export const InlineIndicator = styled.div<{
+  value: boolean
```

Comment:
This `value` should be renamed because it's unclear from the prop name what the value represents / what it does. From my understanding, the `value` is used to determine whether the text inside this styled div should be highlighted to indicate something special about it. I think maybe `shouldIndicate` could work here, or something along those lines?

Reply:
fixed